docs/features/techdocs/how-to-guides.md

```diff
@@ -139,21 +139,25 @@ import { TechDocsCustomHome } from '@backstage/plugin-techdocs';
 
 const techDocsTabsConfig = [
   {
-    label: "Recommended Documentation",
+    label: 'Recommended Documentation',
     panels: [
       {
         title: 'Golden Path',
         description: 'Documentation about standards to follow',
         panelType: 'DocsCardGrid',
-        filterPredicate: entity => entity?.metadata?.tags?.includes('recommended') ?? false,
-      }
-    ]
-  }
-]
+        filterPredicate: entity =>
+          entity?.metadata?.tags?.includes('recommended') ?? false,
+      },
+    ],
+  },
+];
 
 const AppRoutes = () => {
   <FlatRoutes>
-    <Route path="/docs" element={<TechDocsCustomHome tabsConfig={techDocsTabsConfig} />}>
```

Comment:
it was this line which was missing a closing `/>` at the end, but the precommit hook helped move it across multiple lines.